### PR TITLE
PIS-17: Add DB connector

### PR DESCRIPTION
**Pull request by kozlowskimaciej**

### CODE REVIEW

File: backend/api/routers/hello.py [hunk 1]
```diff
@@ -11,5 +12,9 @@
 
 
 @router.get('/')
```
Comment (kubao503) [inline backend/api/routers/hello.py:14]: Why are you using hello router and not some more meaningful one?

**Review by kubao503 — CHANGES_REQUESTED** [review]

File: backend/api/routers/hello.py (hunk 1 above)
Comment (kozlowskimaciej) [reply backend/api/routers/hello.py:14]: Cause that's the only one we've got

Comment (kubao503) [reply backend/api/routers/hello.py:14]: How about you add some new one or change the name of this one?

**Comment by kubao503** [comment]
This DB connector doesn't account for any of the domain model gateways. What is your concept for connecting it to the domain model? Another connector to the db connector?

**Comment by kubao503** [comment]
But domain model is part of the backend

**Comment by kozlowskimaciej** [comment]
But the domain is going to be used in endpoints

**Review by kubao503 — APPROVED** [review]

**Pull request merged**